MAUI/ImageEditor/migration.md

```diff
@@ -528,3 +528,7 @@ Applicable for crop view selection, shape and text annotation selection.
 * Support to add custom shapes or views.
 * Support for pan and zoom improvements while performing the crop action on the images.
 * Support to toolbar customization.
+
+## Known issues
+* Image not loading from stream or URI binding source.

```

Comment:
Image not loading properly from stream or URI binding source.

Reply:
Updated.